tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -360,6 +362,16 @@ def test__parse_args_wrapper_args(self) -> None:
             assert new_arg not in runner.lgbm_kwargs
             assert new_arg in runner.auto_options
 
+    def test_default_arguments(self) -> None:
+        tuner_kwargs = signature(LightGBMTuner).parameters
+        for lgb_train_arg, lgb_train_val in signature(lightgbm.train).parameters.items():
+            if lgb_train_arg in tuner_kwargs:
+                assert tuner_kwargs[lgb_train_arg].default == lgb_train_val.default, (
+                    f"LightGBMTuner '{lgb_train_arg}' default "
+                    "argument is not consistent with original "
+                    "LightGBM 'train' method default argument."
+                )
+
```

Comment:
I talked with some committers including @g-votte @hvy and @Crissman, and we came up with the approach to remove the hard-coded default values.

`LightGBMTuner` and `LightGBMTunerCV` have two types of arguments:
- Arguments with intentionally altered default values like `num_boost_round`: They should be hard-coded.
- Arguments with the same default values with the original LightGBM: we might be able to remove hard-coded default values and just pass through the arguments to the LightGBM functions.

If we can remove the hard-coded default values, we do not need this test case. So, how about focusing on the default value change of `return_cvbooster` and removing the new test cases in this PR?

Reply:
Sure! I will remove the tests. 

Should I leave this: https://github.com/optuna/optuna/pull/3070#issuecomment-955964080 ?

Reply:
Yes!